dlr/dlr-engine/build.gradle

```diff
@@ -8,6 +8,12 @@ repositories {
 
 dependencies {
     api project(":api")
+    testImplementation("org.testng:testng:${testng_version}") {
+        exclude group: "junit", module: "junit"
+    }
+    testImplementation "org.slf4j:slf4j-simple:${slf4j_version}"
+    testRuntimeOnly project(":pytorch:pytorch-engine")
```

Comment:
Why import pytorch? Can't DLR just be used alone? If you are looking for adding Hybrid mode, you can always add it back once you finish the Hybrid functionalities

Reply:
This is for testing. DLR can't be run alone.

Reply:
In this PR, I added the end2end test so it should come with secondary engine